controllers/util/solr_util.go

```diff
@@ -1390,12 +1434,13 @@ func generateSecurityJson(solrCloud *solr.SolrCloud) map[string][]byte {
           %s,
           { "name": "k8s-status", "role":"k8s", "collection": null, "path":"/admin/collections" },
           { "name": "k8s-metrics", "role":"k8s", "collection": null, "path":"/admin/metrics" },
+          { "name": "k8s-zk", "role":"k8s", "collection": null, "path":"/admin/zookeeper/status" },
```

Comment:
You mind if we fix these in a separate small PR?

Reply:
definitely, thanks for catching ... just a quick hack I made for testing on 8.9 but should not be in this PR